src/javascript/app/pages/user/metatrader/metatrader.ui.js

```diff
@@ -186,6 +186,9 @@ const MetaTraderUI = (() => {
                 };
                 $(this).html(typeof mapping[key] === 'function' ? mapping[key]() : info);
             });
+
+            setCounterpartyAndJurisdictionTooltip($('.acc-info div[data="login"]'), acc_type);
```

Comment:
better store `$('.acc-info div[data="login"]')` as a variable outside the map so we don't keep fetching the same element every time